src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -223,6 +326,14 @@ const WorkflowView = _.flow(
             JobHistory.flagNewSubmission(submissionId)
             Nav.goToPath('workspace-job-history', workspaceId)
           }
+        }),
+        variableSelected && h(BucketContentModal, {
+          workspace,
+          onDismiss: () => this.setState({ variableSelected: undefined }),
+          onSelect: v => {
+            this.setState(_.set(['modifiedConfig', 'inputs', variableSelected], v))
+            this.setState({ variableSelected: undefined })
```

Comment:
nit: These two lines can be combined, if you pull `modifiedConfig` out of the `_.set` path and make it explicit.